### PR TITLE
feat: add phone number normalizer

### PR DESCRIPTION
**Pull request by mediv0**
Discussed here https://github.com/persian-tools/persian-tools/discussions/183

tests & docs included

### CODE REVIEW

**Comment by mediv0** [comment]
should i make pull request to develop branch ?
my local repo is based of master need to rebase

**Comment by ghost** [comment]
### CodeSee Review Map:
[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=184&src=https%3A%2F%2Fgithub.com%2Fpersian-tools%2Fpersian-tools)

Review [in an interactive map](https://app.codesee.io/r/reviews?pr=184&src=https%3A%2F%2Fgithub.com%2Fpersian-tools%2Fpersian-tools)

View [more CodeSee Maps](https://app.codesee.io/r/maps?src=https%3A%2F%2Fgithub.com%2Fpersian-tools%2Fpersian-tools)

#### Legend
CodeSee Map Legend

File: README.md [hunk 1]
```diff
@@ -493,6 +493,18 @@ phoneNumberValidator("9022002580"); // true
 phoneNumberValidator("09802002580"); // false
 ```
 
+- Normalizing phone number
+
+```js
+import { normalizePhoneNumber } from "@persian-tools/persian-tools";
+
+phoneNumberNormalizer("+989022002580", "0"); // 09022002580
```
Comment (ali-master) [inline README.md:501]: OMG, My personal phone number is everywhere ^_^

File: README.md [hunk 2]
```diff
@@ -493,6 +493,18 @@ phoneNumberValidator("9022002580"); // true
...
+phoneNumberNormalizer("989022002580", "0"); // 09022002580
+phoneNumberNormalizer("09022002580", "0"); // 09022002580
+phoneNumberNormalizer("09022002580", "+98"); // +989022002580
+phoneNumberNormalizer("09802002580", "0"); // Error: phone number is not valid
```
Comment (ali-master) [inline README.md]: Please remove this sample because there is no need to write error base examples in README.

**Review by ali-master — CHANGES_REQUESTED** [review]
Ready to Merge after fixing an issue.

**Comment by ali-master** [comment]
Thanks @mediv0 Jan.

**Pull request merged**